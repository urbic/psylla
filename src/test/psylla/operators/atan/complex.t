1 1 complex atan ?
