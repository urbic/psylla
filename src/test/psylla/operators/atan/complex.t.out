1.0172219678978514+0.4023594781085251i
